snomed/com.b2international.snowowl.snomed.reasoner/src/com/b2international/snowowl/snomed/reasoner/request/SaveJobRequest.java

```diff
@@ -87,7 +87,6 @@
 	private static final Logger LOG = LoggerFactory.getLogger("reasoner");
 
 	private static final int SCROLL_LIMIT = 10_000;
```

Comment:
Now that `SearchResourceRequestIterator` relies on the revision system for snapshot semantics, limit constants like the above could be changed to eg. `PAGE_SIZE` or similar.